cmd/documentation/exchanges_templates/deribit.tmpl

```diff
@@ -32,7 +32,7 @@ var d exchange.IBotExchange
 
 for i := range bot.Exchanges {
 	if bot.Exchanges[i].GetName() == "Deribit" {
-		c = bot.Exchanges[i]
+		d = bot.Exchanges[i]
```

Comment:
The readme was not regenerated via the documentation tool as suggested in [nit](https://github.com/thrasher-corp/gocryptotrader/pull/1082#discussion_r1062099087)